src/server/src/projects/services/projects.service.ts

```diff
@@ -51,4 +56,13 @@ export class ProjectsService extends TypeOrmCrudService<Project> {
       : builder;
     return paginate<Project>(builderWithFilter, options);
   }
+
+  async findAllUserProjectsPaginated(
+    userId: string,
+    options: AllProjectsOptions
+  ): Promise<Pagination<Project>> {
+    const builder = this.getProjectsBase().andWhere("user.id = :userId", { userId });
```

Comment:
This _also_ needs to filter out `archived` projects.
```suggestion
    const builder = this.getProjectsBase().andWhere(
      "project.archived = FALSE AND user.id = :userId",
      { userId }
    );
```